src/Domain/Fixture/FixtureCreator.php

```diff
@@ -223,6 +251,27 @@ private function createComposerProject(): void {
     $this->codebaseCreator->create($this->options);
   }
 
+  /**
+   * Configures composer "composer-exit-on-patch-failure" parameter.
+   */
+  private function configureComposerExitOnPatchFailure(): void {
+    if (!$this->env->get('ORCA_IS_ALLOWED_FAILURE')) {
+      return;
+    }
+
+    $this->output->section("Setting composer-exit-on-patch-failure to false.");
+    try {
+      $this->composer->setConfig([
+        'extra.composer-exit-on-patch-failure',
+        'false',
+        '--json',
+      ]);
+    }
+    catch (\Exception $e) {
+      $this->output->writeln("Failed to remove Composer platform requirements.");
```

Comment:
This message looks like a copy/paste leftover. Maybe it's supposed to be something like this?

```suggestion
      $this->output->writeln("Failed to set composer-exit-on-patch-failure.");
```